exec/jdbc/src/main/java/org/apache/drill/jdbc/DrillConnection.java

```diff
@@ -213,4 +213,5 @@ void setNetworkTimeout( Executor executor, int milliseconds )
 
   DrillClient getClient();
 
+  boolean useAnsiQuotedIdentifiers() throws SQLException;
```

Comment:
it probably be moved to DrillDatabaseMetadataImpl directly (where other metadata requests are made)